bin/anycablebility

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require 'bundler/setup'
+require 'docopt'
```

Comment:
BTW the main Ossert application is to aggregate several metrics to come up to a more balanced grade of maintenance quality. So it is not very good to use one isolated metric to make your decision.

Anyway I'll try to find better name for "Active Years Age" in order to frustrate nobody.